src/Core/Framework/Plugin/Changelog/ChangelogParser.php

```diff
@@ -16,7 +16,7 @@ public function parseChangelog(string $path): array
         $currentRelease = null;
 
         foreach ($this->parse($path) as $line) {
-            switch ($line[0]) {
+            switch (ltrim($line)[0]) {
```

Comment:
This will break with current changelogs with empty lines ( Uninitialized string offset: 0)

Reply:
Yikes, good hint! So I could either trim in `parse` or use `mb_substr` to get the first char. Any preferences on that?